src/app/custom/translations.tsv

```diff
@@ -720,14 +720,14 @@
 "dataset_edit_enrichment_title"	"Enrichment %{column_name} for row %{row_name}"	"Enrichissement %{column_name} pour la ligne %{row_name}"
 "dataset_edit_success"	"Field updated with value"	"Champ mis à jour avec la nouvelle valeur"
 "header_name_with_errors"	"%{name} (%{errorCount} errors)"	"%{name} (%{errorCount} erreurs)"
-"ws_post-v1-rnsr-json_title"	"Affiliation to RNSR (using rules)"	"Affiliation vers RNSR (par règle)"
-"ws_post-v1-rnsr-json_description"	"Finds the identifier of the National Repository of Research Structures for an affiliation address (with name of the structure)."	"Trouve l’identifiant du Référentiel Nationale des Structure de Recherche pour une adresse d’affiliation (avec nom de la structure)."
-"ws_post-v1-halauthorid-idref-json_title"	"halAuthorId to idRef"	"halAuthorId vers idRef"
-"ws_post-v1-halauthorid-idref-json_description"	"Gives the idRef corresponding to a halAuthorID."	"Donne l’idRef correspondant à un halAuthorID."
-"ws_post-v1-homogenize-document-type-json_title"	"Homogenize document type"	"Homogénéisation du type de document"
-"ws_post-v1-homogenize-document-type-json_description"	"Brings different forms of writing from the document type to a standardized form."	"Ramène différentes formes d’écriture du type de document à une forme standardisée."
-"ws_post-v1-homogenize-publisher-json_title"	"Homogenize publisher"	"Homogénéisation de l’éditeur"
-"ws_post-v1-homogenize-publisher-json_description"	"Brings different forms of writing from the publisher to a standardized form."	"Ramène différentes formes d’écriture d'un éditeur à une forme standardisée."
+"ws_post-v1-rnsr-json_title"	"Assigning RNSR identifier(s) to an address (Alignments)"	"Attribution d’identifiant(s) RNSR à une adresse (Alignements)"
+"ws_post-v1-rnsr-json_description"	"Assigns, using rules, one or more RNSR identifiers based on an author affiliation address and a year of publication."	"Attribue, à l’aide de règles, un ou plusieurs identifiants RNSR à partir d’une adresse d’affiliation d’auteur et d’une année de publication."
+"ws_post-v1-halauthorid-idref-json_title"	"Associate an Author IDREF with a HAL Author ID"	"Associer un IDREF Auteur à un identifiant HAL Author ID"
+"ws_post-v1-halauthorid-idref-json_description"	"Allows to make correspond the identifier IDREF authors to that of the publishers in HAL, having an identifier IDAHL authors."	"Permet de faire correspondre l’identifiant IDREF auteurs à celui des publiants dans HAL, disposant d’un identifiant IDAHL authors."
```

Comment:
<!-- Typo comments are like todo:, where the main issue is a mispelling.  -->
**typo:** ‏
```suggestion
"ws_post-v1-halauthorid-idref-json_description"	"Allows to make correspond the identifier IDREF authors to that of the publishers in HAL, having an identifier IDHAL authors."	"Permet de faire correspondre l’identifiant IDREF auteurs à celui des publiants dans HAL, disposant d’un identifiant IDHAL authors."
```

Reply:
<!-- Polish comments are like a suggestion, where there is nothing necessarily wrong with the relevant content, there's just some ways to immediately improve the quality.  -->
**polish:** ‏English translation... feels not idiomatic

![image](https://github.com/Inist-CNRS/lodex/assets/595509/1dd0e8f9-e9fa-4c69-ac67-13ec52d4f31a)